src/components/HomepageVideo/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import clsx from 'clsx';
+import styles from './styles.module.css';
+
+export function HomepageVideo() {
+  return (
+    <section className={styles.section}>
+      <div className="container">
+        <div className={clsx(`row ${styles.videoSection}`)}>
+          <video width="100%" height="auto" controls>
+            <source src="@site/static/video/earth.mp4" type="video/mp4" />
```

Comment:
I know this is just the demo video, but this URL is incorrect. Based on local testing/[the docusaurus docs](https://docusaurus.io/docs/static-assets), the correct URL is:

```suggestion
            <source src="video/earth.mp4" type="video/mp4" />
```

Reply:
Good catch! Looks like thats why this wasn't loading. Fixed!